pages/code-review.md

```diff
@@ -24,10 +24,34 @@ goals when reviewing code are to:
 - cultivate the ability to critique one’s code more strongly;
 - encourage open communication between the entire team.
 
+## Discuss code review with your team
+
+Every individual and team has a different set of expectations around code
+review. Discussing these expectations explicitly is the best way to make sure
+that everyone is on the same page.
+
+Bonus points for documenting and/or automating the outcomes of your discussion.
```

Comment:
My take on this is that if a team is going to go through this exercise, then there should be an artifact of it that any team member can see and comment on.  Having the discussion to set expectations explicitly is the best way to ensure folks in a team are on the same page.  Documenting the outcome of that discussion(s) and circulating it among team members or posting it publicly is the best way for the team to be able to hold itself accountable to those decisions and expectations.  It also affords the ability to amend them as necessary when circumstances warrant it.  This is especially important for reducing friction of new members joining a team; documenting the decisions/process and putting it somewhere that folks can easily see helps immensely with onboarding newcomers to a project as well as to the team itself.

Now, to @jmcarp's point from our last meeting, if a team can go the extra step and automate the outcomes of the discussion, I completely agree that they should, or at least ought to consider doing so. :-)  Automation helps mitigate the potential to make mistakes (as we are all wont to do!) and serves to maintain a consistent baseline of quality that adheres to the expectations as defined by the team.